lib/controller/StudyRequestController.js

```diff
@@ -34,7 +38,7 @@ StudyRequestController.push({
       schema: StudyRequest.read,
     },
     validate: {
-      payload: StudyRequest.transient,
+      payload: StudyRequest.create,
```

Comment:
hmmm.  this probably means it wasn't validating beforehand, which means that we should double-check that our study request REST API endpoints still work after this change.

(we can test that in one sweep later, though.)